src/sentry/static/sentry/app/views/organizationEventsV2/linkedIssuePreview.tsx

```diff
@@ -3,7 +3,10 @@ import styled from 'react-emotion';
 import PropTypes from 'prop-types';
 
 import {t} from 'app/locale';
-import AsyncComponent from 'app/components/asyncComponent';
+import AsyncComponent, {
+  AsyncComponentProps,
+  AsyncComponentState,
+} from 'app/components/asyncComponent';
```

Comment:
Use `AsyncComponent['props']` and `AsyncComponent['state']` lookup types

Reply:
done.